releases/istio-gatekeeper.yaml

```diff
@@ -0,0 +1,303 @@
+      name: "istio-gatekeeper"
+    livenessProbe:
+      initialDelaySeconds: 10
+      periodSeconds: 30
+      timeoutSeconds: 1
+      failureThreshold: 5
+    readinessProbe:
+      initialDelaySeconds: 10
+      periodSeconds: 30
+      timeoutSeconds: 1
+    resources:
+      limits:
+        cpu: "100m"
+        memory: "128Mi"
+      requests:
+        cpu: "10m"
+        memory: "10Mi"
+
+  ###############################################################################
+  ## Ingress ####################################################################
+  ## Use Kubernetes raw chart to create a OIDC Istio Gateway                    #
+  ## https://github.com/helm/charts/tree/master/incubator/raw                   #
+  ###############################################################################
+- name: 'istio-gatekeeper-additional'
+  chart: "kubernetes-incubator/raw"
+  namespace: "istio-system"
+  labels:
+    component: "iap"
+    namespace: "istio-system"
+    default: "false"
+  version: "0.1.0"
+  wait: true
+  force: true
+  recreatePods: true
+  installed: {{ env "ISTIO_GATEKEEPER_INSTALLED" | default "false" }}
+  values:
+  - resources:
+    - apiVersion: networking.istio.io/v1alpha3
+      kind: Gateway
+      metadata:
+        name: istio-oidc-ingressgateway
+      spec:
+        selector:
+          istio: istio-oidc-ingressgateway
+        servers:
+          - hosts:
+              - "*"
+            port:
+              name: http
+              number: 80
+              protocol: HTTP
+
+  ##############################################################################
+  ## Forecastle ################################################################
+  ## Forecastle is a dynamic web portal for Kubernetes services.              ##
+  ## Used here to provide a page listing all the components deployed behind   ##
+  ## the OIDC ingress.                                                        ##
+  ##############################################################################
+  #
+  # References:
+  #   - https://github.com/stakater/Forecastle/tree/v1.0.27/deployments/kubernetes/chart/forecastle
+  # v1.0.27  url: "git+https://github.com/stakater/Forecastle@deployments/kubernetes/chart?ref=ffb5179aa659e0e3cf0ca20e1c8bd94c5fd66a2e"
+  #   - https://github.com/stakater/Forecastle
+  #
+- name: "istio-gatekeeper-forecastle"
+  namespace: "istio-system"
+  labels:
+    chart: "forecastle"
+    repo: "github"
+    component: "iap"
+    namespace: "istio-system"
+    vendor: "stakater"
+    default: "false"
+  chart: "forecastle/forecastle"
+  version: "v1.0.27"
+  wait: false
+  installed: {{ env "ISTIO_GATEKEEPER_FORECASTLE_INSTALLED" | default "true" }}
+  force: true
+  recreatePods: true
+  values:
+    - nameOverride: istio-gatekeeper-forecastle
+      fullNameOverride: istio-gatekeeper-forecastle
+      forecastle:
+        image:
+          pullPolicy: "IfNotPresent"
+        namespace: "oidc-ingress"
```

Comment:
Namespace should be configurable, and in any case should match the chart's namespace